fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -401,11 +401,16 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 
 		state std::vector<ISimulator::ProcessInfo*>	killProcArray;
 		state std::vector<AddressExclusion>	toKillArray;
+		state std::vector<AddressExclusion>	toKillMarkFailedArray;
 
 		std::copy(toKill.begin(), toKill.end(), std::back_inserter(toKillArray));
 		killProcArray = self->getProcesses(toKill);
+		if (toKillArray.size()) {
```

Comment:
Does the current version only allow mark 1 server failed?
(It is ok to allow only 1 server failed in fdbcli. The test has to be consistent with what is allowed in fdbcli though.)

Reply:
No, fdbcli allows several servers to be mark failed at once (it first performs a safety check on the client's side). I will adjust the test.